apps/client/src/pages/DevHomePage.js

```diff
@@ -1,37 +1,109 @@
+            te gebruiken.
+          </strong>
+        </Alert>
+      </p>
       <table>
-        <thead>
-          <tr>
-            <th>Flow</th>
-            <th>slug</th>
-          </tr>
-        </thead>
+        <tr>
+          <td>
+            <strong>Name</strong>
+          </td>
+          <td>
+            <strong>Flow</strong>
+          </td>
+          <td>
+            <strong>Folder</strong>
+          </td>
+          <td>
+            <strong>#&nbsp;permits</strong>
+          </td>
+        </tr>
         <tbody>
-          {topics
-            .sort((a) => !!a.sttrFile && -1) // make sure sttr flow is on top
-            .map(({ slug, sttrFile, redirectToOlo }) => (
+          {topicsJson
+            .filter(({ hasSTTR }) => !hasSTTR) // only show olo / redir-olo topics
+            .map(({ slug, name, redirectToOlo }) => (
               <tr key={slug}>
                 <td>
-                  {redirectToOlo ? "Redirect" : sttrFile ? "Checker" : "OLO"}
-                </td>
-                <td>
-                  <Link to={geturl(routes.intro, { slug })}>{slug}</Link>
+                  <Link to={geturl(routes.intro, { slug })}>{name}</Link>
                 </td>
+                <td>{redirectToOlo ? "Redirect" : "OLO"}</td>
+                <td>n.a.</td>
+                <td>0</td>
               </tr>
             ))}
+          {topicsJson.map((apiConfig) => {
+            return apiConfig.map((apiTopic) => {
+              const sttrTopic = topicsJson.find(
+                (topic) => topic.slug === apiTopic.slug
+              );
+
+              const title = sttrTopic
+                ? sttrTopic.name
+                : apiTopic
+                ? apiTopic.name || apiTopic.slug
+                : "[ERROR]";
+              return (
+                <tr>
+                  <td>
+                    {sttrTopic && !sttrTopic.hasSTTR ? (
+                      <>
+                        {title}
+                        <br />
+                        <div
+                          style={{
+                            backgroundColor: "rgb(246, 201, 72)",
+                            padding: "1em",
+                          }}
+                        >
+                          STTR file found for <strong>{sttrTopic.name}</strong>,
+                          but we can't load '<strong>{sttrTopic.slug}</strong>'
+                          because it's configured to be{" "}
+                          <strong>
+                            {sttrTopic.redirectToOlo ? "redirectToOlo" : "olo"}
+                            -flow
+                          </strong>
+                          .
+                        </div>
+                      </>
+                    ) : (
+                      <Link to={geturl(routes.intro, apiTopic)}>{title}</Link>
+                    )}
+                  </td>
+                  <td>{sttrTopic ? "configured" : "unknown"}</td>
```

Comment:
"unknown" could be "imported" or "single", right?

Reply:
Good catch. I'll call it 'dynamic' 👍